yesod-test/yesod-test.cabal

```diff
@@ -1,5 +1,5 @@
 name:               yesod-test
-version:            1.6.9.1
+version:            1.6.9.2
```

Comment:
This is my only quibble on the whole thing. It may be better to make this a minor version bump (1.6.10.0) since it has user facing behavioral changes. But I honestly don't feel strongly about this. Feel free to merge/release as you see fit.

Reply:
I'm fine with that, and it also exposes new APIs